src/components/CardList/CardList.jsx

```diff
@@ -0,0 +1,29 @@
+import { Flex, Grid } from '@chakra-ui/layout';
+import React, { useEffect, useState } from 'react'
+import Card from '../Card/Card'
+
+const CardList = () => {
+    const [data, setData] = useState();
+
+    const fetchData = async () => {
+        const res = await fetch("https://gist.githubusercontent.com/jparciga/1d4dd34fb06ba74237f8966e2e777ff5/raw/f3af25f1505deb67e2cc9ee625a633f24d8983ff/youtube-videos-mock.json");
+        const result = await res.json();
+        setData(result);
+    }
```

Comment:
You are fetching data from the gist instead of having a copy in the project, this is good 👍
and also some semicolons 😬
```suggestion
        const res = await fetch("https://gist.githubusercontent.com/jparciga/1d4dd34fb06ba74237f8966e2e777ff5/raw/f3af25f1505deb67e2cc9ee625a633f24d8983ff/youtube-videos-mock.json")
        const result = await res.json()
        setData(result)
    }
```